project-base/src/Shopsys/ShopBundle/Resources/scripts/frontend/window.js

```diff
@@ -165,6 +171,7 @@
         }
 
         Shopsys.register.registerNewContent($window);
+        $window.load();
```

Comment:
To be honest, I'm not sure why are you calling the method. I'm not even sure which method it actually is, I found 2 `.load()` methods in jQuery (https://api.jquery.com/load/ and deprecated https://api.jquery.com/load-event/), both require at least one argument.